backend/src/main/java/com/crm/dto/mapper/AccountMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.crm.dto.mapper;
+
+import com.crm.dto.response.AccountResponse;
+import com.crm.model.db.AccountEntity;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class AccountMapper {
+
+    private final ModelMapper modelMapper;
+
+    public AccountResponse convertToDto(final AccountEntity accountEntity) {
+        return modelMapper.map(accountEntity, AccountResponse.class);
```

Comment:
W przyszłości musimy pomyśleć nad tym aby pozbyć się tych mapperów. Lepszym rozwiązaniem będzie ręczne pisanie mapperów (nie zawsze będzie potrzeba by przypisywać wszystkie wartości z jednego obiektu do drugiego lub nie wszystkie pola będą zgodne nazwą.) Na ten moment może to zostać